provisioning/miniprov/hare_mp/main.py

```diff
@@ -601,6 +601,18 @@ def generate_support_bundle(args):
         if args.t:
             cmd.append('-t')
             cmd.append(args.t[0])
+        if args.duration:
+            noop(args.duration)
+        if args.size_limit:
+            noop(args.size_limit)
+        if args.service:
```

Comment:
should it be "services" instead of "service"?

Reply:
Yes. Done.